pkg/network/node/vxlan_monitor_test.go

```diff
@@ -211,15 +218,56 @@ func TestEgressVXLANMonitor(t *testing.T) {
 
 	packetsIn(ovsif, inCounts, "192.168.1.5")
 	retry = evm.check(false)
-	if update := peekUpdate(updates); update == nil {
+	if update := peekUpdate(updates, evm); update == nil {
 		t.Fatalf("Node failed to recover")
-	} else if update.nodeIP != "192.168.1.5" || update.offline {
+	} else if update[0].nodeIP != "192.168.1.5" || update[0].offline {
 		t.Fatalf("Unexpected updated node %#v", update)
+	} else if len(update) > 1 {
+		t.Fatalf("Check erroneously showed additional updated nodes %#v", update)
 	}
-	if update := peekUpdate(updates); update != nil {
+
+	if update := peekUpdate(updates, evm); update != nil {
 		t.Fatalf("Check erroneously showed additional updated node %#v", update)
 	}
 	if retry {
 		t.Fatalf("Check erroneously requested retry")
 	}
+
+	// evm.check doesn't block when evm.updates is full.
+	// https://bugzilla.redhat.com/show_bug.cgi?id=1824203
```

Comment:
"Ensure evm.check doesn't block ...". The way it is now it looks like you're just quoting the bug description, which would imply the desired behavior was the opposite of that